dao/src/test/java/cz/idomatojde/TimetableChatMessageTest.java

```diff
@@ -225,12 +225,46 @@ public void deleteTest() {
         chat.setSender(offer.getOwner());
         chat.setTimetableEntry(entry);
 
-        chatMessagesDAO.create(chat);
+        timetableChatMessageDAO.create(chat);
+
+        // Act
+        timetableChatMessageDAO.delete(chat);
+
+        // Assert
+        timetableChatMessageDAO.getById(chat.getId());
+    }
+
+    @Test
+    public void getAllMessagesOfUser(){
+        // Arrange
+        var offer = getOffer("get all messages");
+        var anotherUser = getUser("another");
+        userDao.create(offer.getOwner());
+        userDao.create(anotherUser);
+        offerDao.create(offer);
+        var timetable = timetableDAO.createTimetable(offer.getOwner(), 2012, 2);
+        var entry = timetableDAO.createEntry(timetable, offer, LocalTime.now(), Duration.ofMinutes(50));
+        var msg = "my message";
+
+        var chat = new TimetableChatMessage();
+        chat.setText(msg);
+        chat.setSender(offer.getOwner());
+        chat.setTimetableEntry(entry);
+
+        var anotherChat = new TimetableChatMessage();
+        anotherChat.setText("text");
+        anotherChat.setSender(anotherUser);
+        anotherChat.setTimetableEntry(entry);
+
+        timetableChatMessageDAO.create(chat);
+        timetableChatMessageDAO.create(anotherChat);
 
         // Act
-        chatMessagesDAO.delete(chat);
+        var messages = timetableChatMessageDAO.getAllMessagesOfUser(offer.getOwner());
 
         // Assert
-        chatMessagesDAO.getById(chat.getId());
+        Assert.assertEquals(messages.size(), 1);
+        Assert.assertEquals(messages.get(0), chat);
```

Comment:
We should really use a single assertion mechanism, and I believe we have more occurences of assertThat() than Assert.assertEquals()

Reply:
Ha, missed that. They will be changed